docs/fields/field-details.asciidoc

```diff
@@ -6414,6 +6430,25 @@ type: keyword
 
 
 
+| extended
+
+// ===============================================================
+
+|
+[[field-orchestrator-resource-ip]]
+<<field-orchestrator-resource-ip, orchestrator.resource.ip>>
+
+| IP address assigned to the resource being acted upon by the orchestrator. In Kubernetes, for instance, this could be the IP of a Pod (as opposed to the Node on which the Pod is running).
+
+type: ip
+
+
+Note: this field should contain an array of values.
+
```

Comment:
should be put a caveat comment for Pods. E.g.   "In the case of Kubernetes Pods, there will be one IP address in this array"

Reply:
This note was auto-generated. I updated the long description to cover that as well.